src/Controllers/ItemController.php

```diff
@@ -58,12 +59,13 @@ public function showItem(
         /** @var TemplateConfigService $templateConfigService */
         $templateConfigService = pluginApp(TemplateConfigService::class);
 
-        if ($variationId > 0 && (int)$templateConfigService->get('item.show_please_select') == 1) {
+        if ($variationId > 0 && (int)$templateConfigService->getInteger('item.show_please_select') == 1) {
```

Comment:
```suggestion
        if ($variationId > 0 && $templateConfigService->getInteger('item.show_please_select') == 1) {
```
->getInteger() sollte schon einen integer-Wert liefern.